apps/hashdotai/docs/hash/working-with-data-in-pages/using-blocks-in-pages.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Using Blocks in Pages
+slug: hash/working-with-data-in-pages/using-blocks-in-pages
+objectId: ???
+description: Using Blocks in Pages
+---
+
+# Using Blocks in Pages {#using-blocks-in-pages}
+
+In order to view, create, or manipulate data or information in HASH, you’ll need to use Blocks.
+
+## How Blocks work with Entities, Links, and Entity Types {#how-blocks-work-with-entities-links-and-entity-types}
+
+Blocks can render and manipulate Entities, Links, Entity Types, and even Property, Data, and Link Types, though this is less common. Blocks can also create new Entities, Links, and Entity Types.
+
+You might use a Block which renders some Entities and Links which already exist in your HASH Workspace. Or you might use a Block to create new Entities of an Entity Type which doesn’t exist yet. Since Blocks are developed by third-parties, there is a huge range of possible data workflows which they enable.
+
+## The default Block: Paragraph Block {#the-default-block-paragraph-block}
+
+When you create a new Page in HASH, a Paragraph Block will automatically be inserted at the top of the page and your cursor will be placed inside the block, ready to type. HASH does this so that Pages can be used to quickly capture some writing.
+
+The Paragraph Block creates a Text Entity in your HASH Workspace. The Text Entity conforms to the Text Entity Type, which is a non-editable system type. As you write in the Paragraph Block, the ‘Tokens’ Property on the Text Entity is updated to capture your text.
+
+Hitting the return key in the Paragraph Block will create a new Paragraph Block below the first one. Each Paragraph on a Page will be its own Block. You can tell this from the [...] context menu on the left of the block, from the comment button on the right of the block, and from the —[+]— button which appears above and below the Paragraph when you hover.
```

Comment:
@jtewright How were you imagining the `—[+]—` would be styled?

Reply:
I was trying to represent that button which is like a horizontal line with plus symbol in the middle. Might be best to just describe it as the [+] button though since the dashes either side are probably confusing.